tests/test_kci.py

```diff
@@ -1,19 +1,22 @@
 import numpy as np
 from flaky import flaky
-from sklearn.metrics.pairwise import linear_kernel, polynomial_kernel, rbf_kernel
+from sklearn.metrics.pairwise import rbf_kernel
 
 from pywhy_stats import kci
 from pywhy_stats.kernels import delta_kernel
 
 ####################################################
 # Unconditional independence tests - Continuous
 ####################################################
+seed = 12345
+rng = np.random.default_rng(seed)
 
 
 @flaky(max_runs=3)
 def test_given_continuous_independent_data_when_perform_kernel_based_test_then_not_reject():
-    x = np.random.randn(1000, 1)
-    y = np.exp(np.random.rand(1000, 1))
+    n_samples = 200
+    x = rng.standard_normal((200, 1))
```

Comment:
Any reason to change it to `rng` instead of using `numpy`? If it is to use the random generator, this is already covered by the flaky decorator of the test, i.e. no need to have a seed.

Reply:
It's quite difficult to debug tests if they're not deterministic, so the nice thing is if we break stuff and this test fails, we can keep running it to get the same answer and do some debugging. With the previous random seed, I spent a lot of time trying to figure out what was going on because the test didn't pass but the output was changing, so hard to isolate.

The flaky thing will still work because the CIs inherently test on different distros/OS/etc., so this will still have the same features, but just locally you can reproduce failures to the exact numerical precision.